README.md

```diff
@@ -170,25 +177,103 @@ end,
 
 ### File types
 
-The options can also be scoped to specific file types. In the default configuration the templates for the `markdown`, `html`, `tex` ..., files override the template defined in the global settings. Any option can be added under the specific file type, not just the template. For instance, if you only want to use absolute file paths for LaTeX, then:
+The options can also be scoped to specific file types.
+In the default configuration the templates for the `markdown`, `html`, `tex` ..., files override the template defined in the global settings.
```

Comment:
Should be comma after "configuration" ;)

Perhaps this is better?

"File-type specific options will override the default (or global) configuration."

In the next line:

"Any option can be specified for a specific file type. For instance, if you only want to use absolute file paths for LaTeX, then: ..."